modules/ROOT/pages/publish-apis.adoc

```diff
@@ -33,12 +33,12 @@ Now set the API information, including visibility, icon, and description.
 . Select *Replace Icon* to set the API version icon.
 . Enter a *Description* for the API version.
 . Leave the *Visibility* as *Admin Only* while you preview the API version as described in the next section.
-. When your API is displayed on the community site in the API carousel component or in an API card, by default it has a single *API Details* button. It can also have an optional *Learn more* button that links to a business content page. In the *Associated "Learn-more" Page* section, select *None* to keep the default, or select *Custom page* and set the *Custom page URL* to a business content page created in the *Community Builder*.
+. When your API is displayed on the community site in the API carousel component or in an API card, by default it has a single *API Details* button. It can also have an optional *Learn more* button that links to a business content page. In the *Associated "Learn-more" Page* section, select *None* to keep the default, or select *Custom page* and set the *Custom page URL* to a business content page created in Community Builder.
 . Select *Apply*.
 
 == Preview APIs
 
-. In the ACM control panel, select *Community Builder*. The Community Builder displays a preview of
+. In the ACM control panel, click *Community Builder*. Community Builder displays a preview of
```

Comment:
Break step after first sentence.
Reconsider providing the expected result; the user is in the UI and sees this, right?